src/classes/frDonation.cls

```diff
@@ -77,10 +77,12 @@ public class frDonation extends frModel {
         this.setDonorId(String.valueOf(request.get('donorId')));
         String status = String.valueOf(request.get('status'));
         
-        if (Boolean.valueOf(request.get('pledge'))) {
+        if(String.isBlank(o.StageName)) {
```

Comment:
So here's how I was testing this issue:
1. Create a new donation via giving form.
2. See that the new donation shows up in Salesforce as an opportunity. Its status is Closed Won.
3. Refund said donation.
4. Check the opportunity in Salesforce again. It should say Closed Lost.

When I do this, I still see it as Closed Won. I'm thinking it might be because once the request goes through again, `String.isBlank(o.StageName)` is no longer blank (because it was assigned when going through this block the first time).

Reply:
Determined this was likely because our default mapping suggestion includes a constant mapping for StageName.  This was a holdover from before we had this managed package where it wasn't customizable and every opportunity was "Closed Won", so the defaults were intended to reproduce that original behavior.  At this point I don't think it makes sense for us to include a default mapping and instead have no mapping for StageName so we hit this logic and this becomes the default behavior.  Will fix

Reply:
Install link that fixes this:
https://login.salesforce.com/packaging/installPackage.apexp?p0=04t1C000000OD0X